tests/infrastructure/vm_console_proxy/conftest.py

```diff
@@ -140,15 +124,24 @@ def vm_service_account_role_binding(namespace, vm_console_proxy_service_account)
 
 
 @pytest.fixture(scope="class")
-def vm_console_proxy_service_account_role_binding(namespace, vm_console_proxy_service_account, vm_console_proxy_role):
+def assert_vm_console_proxy_cluster_role():
```

Comment:
no verbs in fixture name please. May be name it `console_proxy_cluster_role` ? Also, return it if it exist, else raise resource not found exception.

Reply:
I just want to use VM_CONSOLE_PROXY_CLUSTER_ROLE in fixture vm_console_proxy_service_account_role_binding so just to be sure it exists.